ui/css/logviewer.css

```diff
@@ -9,17 +9,56 @@ body {
     bottom: 0;
     left: 0;
     min-width: 240px;
-    padding-top: 10px;
     display: flex;
     flex-flow: column;
 }
 
+/* Boostrap navbar overrides for logviewer */
+.navbar {
+    min-height: 41px; /* 40 for contained elements + lower bootstrap border */
+    border-top: 0; /* Shut off the top border */
+    border-radius: 0; /* Straight navbar */
+    margin-bottom: 10px; /* Breathing space for run-data */
+}
+
+.navbar-nav > li > a {
+    padding: 10px 15px;
+    font-size: 12px; /* Actionbar contents */
+}
+
+.navbar-nav > li > div {
+    padding: 10px 15px;
+    line-height: 20px;
+}
+
+.navbar-nav > li > span {
+    line-height: 20px;
+}
```

Comment:
So this stuff from L24-L36 works faultlessly at all browser dimensions and zoom values. I did try to simplify their logic further but the markup configuration is a bit inter-dependent with how these get applied. I can have at it some more if needed :)